mule-deployer/src/main/java/org/mule/tools/client/standalone/controller/MuleProcessController.java

```diff
@@ -15,17 +15,15 @@
 
 public class MuleProcessController {
 
-  private static final int DEFAULT_TIMEOUT = 60000;
+  private static final String DEFAULT_TIMEOUT = "60000";
+  private static final int controllerTimeout =
+      Integer.parseInt(System.getProperty("mule.process.controller.timeout", DEFAULT_TIMEOUT));
```

Comment:
please change this, if the value is a non parseable number it will fail. 
Better do a private getControllerTimeout 

Also, I reckon you'll remove this but please remember private static final should be ALL_CAPPS ;)

Reply:
Ok!